KomaMRIBase/src/sequences/RF/design.jl

```diff
@@ -0,0 +1,152 @@
+"""
+"""
+function block_pulse(flip_angle; duration=0, freq_offset=0, phase_offset=0,
+    time_bw_product=0, bandwidth=0, delay=0,
+    dead_time=0, ring_down_time=0, sys=nothing)
+
+    if !isnothing(sys)
+        dead_time = sys.RF_dead_time_T
+        ring_down_time = sys.RF_ring_down_T
+    end
+
+    if duration == 0
+        if time_bw_product > 0
+            duration = time_bw_product / bandwidth
+        elseif bandwidth > 0
+            duration = 1 / (4 * bandwidth)
+        else
+            @error "Either bandwidth or duration must be defined"
+        end
+    end
+    amplitude = flip_angle / (2π * γ * duration) * cis(phase_offset)
+
+    if dead_time > delay
+        delay = dead_time
+    end
+
+    dly = Delay(0)
+    if ring_down_time > 0
+        dly = Delay(delay + duration + ring_down_time)     # I NEED a review
+    end
+
+    return RF(amplitude, duration, freq_offset, delay), dly
```

Comment:
it should output a Sequence

Reply:
The idea now is to add a duration to the ouput Sequence

Reply:
seq.DUR needs to be a vector of Delay's. To be able to combine them like we talked #305